textproc/jtc/Portfile

```diff
@@ -37,6 +37,8 @@ build.cmd           ${configure.cxx}
 build.args          -o jtc -Wall -std=c++14 -Ofast
 build.target        jtc.cpp
 
+compiler.cxx_standard 2014
```

Comment:
Needs to be set before you use the value of ${configure.cxx}.

Reply:
Fixed